src/CMakeLists.txt

```diff
@@ -6,7 +6,12 @@ add_library(asm-parser-lib STATIC
     objdump/parser.cpp
     assembly/parser.cpp
 )
-target_link_libraries(asm-parser-lib PUBLIC ${CONAN_LIBS})
```

Comment:
With the new integrations, there are is 0 need for the cmakelist to know about conan, which I like _a lot_